client/src/reducers/genesetsUI.js

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
this reducer needs a unit test in `client/__tests__/reducer/`

Reply:
Yup, incoming